client/js/views/LoginSignup.vue

```diff
@@ -196,7 +199,7 @@ export default {
   justify-content: center;
   margin-top: 16px;
 }
-@media (orientation: landscape) and (max-width: 979px) {
+@media (orientation: landscape) and (max-width: 960px) {
```

Comment:
Was this adjustment intentional? Any reason for the switch? Should it be aligned with anywhere else on the site? It feels like you should normalize breakpoints and have some css utilities to call for breakpoint stuff but it may not be that important right now.

Reply:
Definitely want to revisit these overall. I changed this here because of the introduction of the nav drawer which is a responsive component and has a breakpoint at 960px, so it looks and feels better for that to align with the media query breakpoint